transport/utils/validate.go

```diff
@@ -20,6 +21,16 @@ func init() {
 	uni := ut.New(english, english)
 	translator, _ = uni.GetTranslator("en")
 	_ = entrans.RegisterDefaultTranslations(Validate, translator)
+
+	Validate.RegisterTagNameFunc(func(fld reflect.StructField) string {
```

Comment:
A bit confused about this change; especially because the `init()` function is not used anywhere.. 🤔

Reply:
Should be called automatically when i import the package.

This change ensures that when validation errors occur, the field names in the error messages correspond to the JSON tags of the fields, which is more user-friendly.